### ISSUE TITLE
Misalignment position of mask on input with `height`

### ISSUE THREAD

**Issue by horihiro** [horihiro/TextBlurrer-ChromeExtension#26]
```html
<input style="min-height: 32px" value="value to be blurred"/><br>
```

![image](https://github.com/horihiro/TextBlurrer-ChromeExtension/assets/4566555/20d2de37-acaa-4c5f-ba59-2b443f4bd6e8)

**Comment by horihiro** [comment]
This issue is happened when `font-size` of an `input` element is smaller then `height` of the element.

**Comment by horihiro** [comment]
Resolved by #28
